bin/i18n/redact_restore_utils.rb

```diff
@@ -7,14 +7,24 @@
 require_relative 'i18n_script_utils'
 
 class RedactRestoreUtils
+  # RedactRestoreUtils manages the redaction and restoration of i18n files.
+  def self.backup_source_file(source_path)
+    return unless File.exist?(source_path)
+    return unless source_path[I18N_SOURCE_DIR]
+    backup_path = source_path.sub(I18N_SOURCE_DIR, I18N_ORIGINAL_DIR)
+    I18nScriptUtils.copy_file(source_path, backup_path)
+  end
+
+  # For a given file, redact_file returns the redacted content of the file. The default file format is Markdown.
   def self.redact_file(source_path, plugins = [], format = 'md')
```

Comment:
Are there no tests for this method?

Reply:
hmm there were no tests before. I'm not sure how to test window commands